clients/cli/src/create_escrow_account.rs

```diff
@@ -0,0 +1,153 @@
+use {
+    crate::common::get_account_owner, solana_keypair::Keypair,
+    spl_token_2022::instruction::initialize_account,
+};
+use {
+    crate::{
+        common::{parse_pubkey, parse_token_program, process_transaction}, /* Keep parse_token_program if we add an override flag later */
+        config::Config,
+        output::{format_output, println_display},
+        CommandResult,
+    },
+    clap::Args,
+    serde_derive::{Deserialize, Serialize},
+    serde_with::{serde_as, DisplayFromStr},
+    solana_cli_output::{display::writeln_name_value, QuietDisplay, VerboseDisplay},
+    solana_program_pack::Pack,
+    solana_pubkey::Pubkey,
+    solana_signature::Signature,
+    solana_signer::Signer,
+    solana_system_interface::instruction::create_account,
+    solana_transaction::Transaction,
+    spl_token_wrap::{get_wrapped_mint_address, get_wrapped_mint_authority},
+    std::fmt::{Display, Formatter},
+};
+
+#[derive(Clone, Debug, Args)]
+#[clap(about = "Creates an escrow token account for holding unwrapped tokens")]
+pub struct CreateEscrowAccountArgs {
+    /// The address of the mint for the unwrapped tokens the escrow will hold
+    #[clap(value_parser = parse_pubkey)]
+    pub unwrapped_mint: Pubkey,
+
+    /// The address of the token program for the *wrapped* mint
+    #[clap(value_parser = parse_token_program)]
+    pub wrapped_token_program: Pubkey,
+}
+
+#[serde_as]
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CreateEscrowAccountOutput {
+    #[serde_as(as = "DisplayFromStr")]
+    pub escrow_account_address: Pubkey,
+
+    #[serde_as(as = "DisplayFromStr")]
+    pub escrow_account_owner: Pubkey, // This is the wrapped_mint_authority PDA
+
+    #[serde_as(as = "DisplayFromStr")]
+    pub escrow_token_program_id: Pubkey,
+
+    #[serde_as(as = "Option<DisplayFromStr>")]
+    pub signature: Option<Signature>,
+}
+
+impl Display for CreateEscrowAccountOutput {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        writeln_name_value(
+            f,
+            "Escrow Account Address:",
+            &self.escrow_account_address.to_string(),
+        )?;
+        writeln_name_value(
+            f,
+            "Escrow Account Owner (PDA):",
```

Comment:
nit: maybe just spell it out totally
```suggestion
            "Escrow Account Owner (wrapped mint authority):",
```